docs/docs/llm_as_judge.rst

```diff
@@ -85,24 +85,32 @@ of metrics.rag.model_response_assessment.llm_as_judge_by_flan_t5_large_on_hf_pip
 .. code-block:: python
 
     from unitxt import add_to_catalog
-    from unitxt.inference import HFPipelineBasedInferenceEngine
+    from unitxt.inference import (
+    IbmGenAiInferenceEngine,
+    IbmGenAiInferenceEngineParams)
     from unitxt.llm_as_judge import LLMAsJudge
 
-    inference_model = HFPipelineBasedInferenceEngine(
-        model_name="google/flan-t5-large", max_new_tokens=32
-    )
-    recipe = (
-        "card=cards.rag.model_response_assessment.llm_as_judge_using_mt_bench_template,"
-        "template=templates.rag.model_response_assessment.llm_as_judge_using_mt_bench_template,"
-        "demos_pool_size=0,"
-        "num_demos=0"
+    model_id = "meta-llama/llama-3-8b-instruct"
+    gen_params = IbmGenAiInferenceEngineParams(max_new_tokens=252)
```

Comment:
The example in the documentation should not be based only on IBMGenAI, because most people can not access it.

Reply:
I'm in the middle of rewriting it